source/deployment/sso-saml-onelogin.rst

```diff
@@ -28,23 +28,41 @@ Create a OneLogin Connection App for Mattermost SSO
 
 2. Configure the app.
   a. Select the **Configuration** tab, then enter the following values:
+    - **RelayState**: leave blank
+    - **Audience**: leave blank
     - **Recipient**: ``https://<your-mattermost-url>/login/sso/saml`` where ``https://<your-mattermost-url>`` should typically match the `Mattermost Site URL <https://docs.mattermost.com/administration/config-settings.html#site-url>`__.
     - **ACS (Consumer) URL Validator**: ``https:\/\/<your-mattermost-url>\/login\/sso\/saml``
     - **ACS (Consumer) URL**: ``https://<your-mattermost-url>/login/sso/saml``
 
   .. image:: ../../source/images/onelogin_3_configuration_1.png
+  
+  b. In System Console,  `enable encryption <https://docs.mattermost.com/administration/prev-config-settings.html#enable-encryption>`__, save your changes, then return to the Configuration tab in access the **SAML Encryption** field.
```

Comment:
@icelander - Your documentation feedback included: "Add note that this field only shows up if you enable Encryption and then Save the application, then go back to the Configuration tab:" -- Please confirm that this documentation update is correct based on your recommendation.

Reply:
That field only shows up in OneLogin if you have encryption enabled for the application. Something like this:

> b. Check `Encrypt Assertion`, then click `Save`. This will redirect you to the `Info` tab, so click `Configuration`

Also, line 41 should read:

> Paste the Public Key that you generated earlier into the **SAML Encryption** field at the bottom of the page.